libcontainer/integration/exec_test.go

```diff
@@ -1660,6 +1661,70 @@ func TestPIDHost(t *testing.T) {
 	}
 }
 
+func TestPIDHostInitProcessWait(t *testing.T) {
+	if testing.Short() {
+		return
+	}
+
+	rootfs, err := newRootfs()
+	ok(t, err)
+	defer remove(rootfs)
+
+	pidns := "/proc/1/ns/pid"
+
+	config := newTemplateConfig(rootfs)
+	config.Namespaces.Add(configs.NEWPID, pidns)
+	container, err := newContainerWithName("test", config)
+	ok(t, err)
+	defer func() {
+		err := container.Destroy()
+		ok(t, err)
+	}()
+
+	// Run a container with two long-running processes.
+	stdinR1, stdinW1, err := os.Pipe()
+	ok(t, err)
+	process1 := &libcontainer.Process{
+		Cwd:   "/",
+		Args:  []string{"cat"},
```

Comment:
Easier to do with something like `sleep 100`, this way you don't need those pipes.

Reply:
I agree. Will post another change.